### PR TITLE
register by date screen forntend in progress

### PR DESCRIPTION
**Pull request by ShirAmit96**

### CODE REVIEW

File: backend/app.js [hunk 1]
```diff
@@ -3,6 +3,11 @@ const app = express();
 app.get('/', function (req, res) {
   res.send('Hello World!');
 });
-app.listen(3000, () => {
+
+app.listen(3001, () => {
```
Comment (kimi-codes) [inline backend/app.js:7]: [In Yasmins PR ](https://github.com/ArnonEilat/kueen-b/pull/38/files) she changes the backend port to be 5000, and the frontend to be 3001.

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+import { Link } from 'react-router-dom';
+
 
 function App() {
+  
   const [name, setName] = React.useState("");
   const [email, setEmail] = React.useState("");
 
+  
+
```
Comment (kimi-codes) [inline frontend/src/App.js:17]: one empty line before the return is enough :)

File: frontend/src/RegisterByDate.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./RegisterByDate.css";
+import Arrow from "./Arrow.svg";
+import { render } from "@testing-library/react";
+
+function goBack() {
+  <link to="/.App.js"></link>;
+}
+function registerToDate() {}
```
Comment (kimi-codes) [inline frontend/src/RegisterByDate.js:9]: registerToDate and registerByDate are very similar names and could be confusing, maybe one of them can be just register/registration or something else? but its up to you, you can keep the names as is

File: backend/app.js [hunk 4]
```diff
@@ -3,6 +3,11 @@ const app = express();
...
   console.log('Example app listening on port 3000!');
+})
+
+app.get('/a', function (req, res) {
+  res.send('Hello!');
```
Comment (kimi-codes) [inline backend/app.js:12]: why do we need this "/a" endpoint?

**Review by katz-tom — COMMENTED** [review]
looks good!

**Review by katz-tom — APPROVED** [review]

**Pull request merged**